<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="commit_btn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>입장</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>292</width>
     <height>71</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>휴먼모음T</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string> IP: </string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit"/>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>휴먼모음T</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>학번: </string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>휴먼모음T</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>이름: </string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_3"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
